source/_integrations/google_maps.markdown

```diff
@@ -28,6 +28,10 @@ device_tracker:
     username: YOUR_USERNAME
 ```
 
+Once enabled and you have rebooted devices discovered through this integration wil be listed in the `known_devices.yaml`file within your configuration directory.
```

Comment:
Did we not already say goodbye to the `known_devices.yaml` file?

Reply:
I'm not aware of such thing. That being said I'm not that actively involved in the core.

Reply:
It is still used.